src/test/java/core/basesyntax/strategy/BalanceOperationHandlerImplTest.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.strategy;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationHandlerImplTest {
+    private static OperationHandler operationHandler;
+    private static StorageDao storageDao;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {

```

Comment:
```suggestion
    public static void beforeClass() {
```
this method is not supposed to throw an exception